client/src/components/CurrentTeam.jsx

```diff
@@ -9,18 +9,18 @@ class CurrentTeam extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      people: this.props.test,
-      currentView: this.props.test.slice(0, 8),
-      total: this.props.test.length,
+      people: [],
+      currentView: null,
+      total: null
```

Comment:
If you had a lot of state to manage and wanted to cut down on state variables, I think you can get rid of 'currentView' and 'total' by using 'people' only. Total would be people.length and currentView would be people.slice(0, 8).

Reply:
That's a very good point! Would definitely cut down on the number of state to manage. I will implement.